tools/firmware_hash.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+from hashlib import blake2s
+from os import sys
```

Comment:
Just `import sys` is the common way, even though this also works, to my surprise